### DISCUSSION TITLE
Is Deep-Live-Cam Unusable for Commercial Purposes Because It Uses the InsightFace Model?

### DISCUSSION THREAD

**Discussion by yhayano-ponotech** [hacksider/Deep-Live-Cam#203] [Q&A]
The readme credits indicate that this code uses the InsightFace model. Does this mean that it cannot be used for commercial purposes? As far as I know, the InsightFace model is intended for non-commercial research purposes only.  I would like to hear your thoughts on this.

**Comment by akkatraffer** [comment]
Сan this software work with multiple video cards? if so, how to configure the program for two or more video cards for optimal operation?

**Reply by hacksider** [reply]
You can use the software/code in any purpose BUT the models are not. There are other alternatives that are free to ise for commercial use. I believe facefusion have implemented those models with those licenses but they are not that good when it comes to real-time deep fakes. 

You are allowed to modify the code as you like as it is based on roop (see the terms of license), And add those implementations on it. You are also free to contribute to this project in case you are successful on it.

**Comment by hacksider** [comment]
You can use the software/code in any purpose BUT the models are not. There are other alternatives that are free to ise for commercial use. I believe facefusion have implemented those models with those licenses but they are not that good when it comes to real-time deep fakes. 

You are allowed to modify the code as you like as it is based on roop (see the terms of license), And add those implementations on it. You are also free to contribute to this project in case you are successful on it.

So to answer the question, the model is for non-commercial use only. Try contacting insightface for the license in case you want to use it for your business.

**Reply by yhayano-ponotech** [reply]
Thank you for your reply. I understood that the models are not commercial use.
